File: c/fizzbuzz1.c

#include <stdio.h>
int main(){
    for(int i=0;i<=100;i++){
        int fzbz = 0;
        if(i % 3 == 0){
            fzbz = 1;
            printf("Fizz");
        }
        if(i % 5 == 0){
            fzbz = 1;
            printf("Buzz");
        }
        if(!fzbz){
            printf("%d",i);
        }
        printf("\n");
    }
    return 0;
}

File: ruby/fizzbuzz1.rb

(0..100).each do |i|
    fzbz = false
    if i % 3 == 0
        fzbz = true
        print "Fizz"
    end
    if i % 5 == 0
        fzbz = true
        print "Buzz"
    end
    if not fzbz
        print i
    end
    puts
end
